CodePush.m

```diff
@@ -42,6 +43,10 @@ + (NSURL *)bundleURLForResource:(NSString *)resourceName
                   withExtension:(NSString *)resourceExtension
 {
     NSError *error;
+    if (launchingApp) {
```

Comment:
Will this work if the install mode is ON_NEXT_RESTART? Because my conjecture is that case this flag will be true on start, hence updates will ALWAYS rollback.